### PR TITLE
add test for creating session

### PR DESCRIPTION
**Pull request by awyyyn**

### CODE REVIEW

File: src/modules/sessions/sessions.route.ts [hunk 1]
```diff
@@ -29,6 +29,17 @@ const controller = new SessionsController();
  * @property {string} updatedAt - updated at
  */
 
+/**
+ * @typedef {object} CreateSession
+ * @property {string} name.required - Name of the session
+ * @property {string} location.required - location
+ * @property {string} sessionTime.required - Time of session
```
Comment (1Domz) [inline src/modules/sessions/sessions.route.ts:37]: please add sessionDate you could base on this

https://github.com/Zeff01/formapp-be/blob/ccf2d30a8f7ffc6937558050b4db50d24ce5b6bd/src/modules/sessions/sessions.service.ts#L102-L106

File: src/modules/sessions/sessions.route.ts [hunk 2]
```diff
@@ -50,7 +61,7 @@ const controller = new SessionsController();
  * @property {integer} noofTeams - Number of teams that will be created
  * @property {integer} maxperTeams - Maximum player of all teams that has been created
  * @property {integer} maxPlayers - Maximum player per teams that will be created
- * @property {'OPENPLAY' | 'TRAINING' | 'TOURNAMENT'} sessionType - Type of session (OPENPLAY, TRAINING, TOURNAMENT)
+ * @property {'OPENPLAY' | 'TRAINING' | 'TOURNAMENT'} \ - Type of session (OPENPLAY, TRAINING, TOURNAMENT)
```
Comment (1Domz) [inline src/modules/sessions/sessions.route.ts]: there is typo here

**Comment by 1Domz** [comment]
This code should be replace to your newly created session 
https://github.com/Zeff01/formapp-be/blob/ccf2d30a8f7ffc6937558050b4db50d24ce5b6bd/src/modules/sessions/sessions.route.ts#L18-L30

and it should be together so it's easy to read like this

`/**
 * Create user body
 * @typedef {object} CreateUserBody
 * @property {string} email.required - email of user
 * @property {string} name.required - name of user
 * @property {string} cognitoId.required - cognito id
 * @property {string} phone - phone number
 */
/**
 * User
 * @typedef {object} User
 * @property {string} email - email of user
 * @property {string} name - name of user
 * @property {string} cognitoId - cognito id
 * @property {string} phone - phone number
 */
/**
 * POST /users/create
 * @summary Create user
 * @tags users
 * @param {CreateUserBody} request.body.required
 * @return {User} 201 - user created
 */
users.post(
  '/create',
  verifyAuthToken,
  RequestValidator.validate(CreateUserDto),
  controller.createUser
);`

**Comment by 1Domz** [comment]
also rename the file name in test it should be `test/module/Session/functionName(folder)/functionName.test.ts`
same as data 

to run these directly right click in the file and click `Open in Integrated Terminal`
and run these command `npm run test functionName.test.ts` it should be succeed first
there is docs for these test if you want to explore more https://github.com/Zeff01/formapp-be/blob/ccf2d30a8f7ffc6937558050b4db50d24ce5b6bd/tests.md and for jest docs https://jestjs.io/docs/getting-started

**Comment by awyyyn** [comment]
```bun
bun run createSession.test.ts
```
### Result
image

**Comment by 1Domz** [comment]
pull from dev too before I merge it thanks.

**Comment by awyyyn** [comment]
done Sir.

**Review by 1Domz — APPROVED** [review]

**Pull request merged**